addons/project/views/project_views.xml

```diff
@@ -1815,10 +1815,12 @@
             </field>
         </record>
 
-        <menuitem name="My Tasks" id="menu_project_management" parent="menu_main_pm"
-            action="action_server_view_my_task" sequence="2" groups="group_project_user"/>
+        <menuitem name="Tasks" id="menu_project_management" parent="menu_main_pm" sequence="2"/>
 
-        <menuitem name="All Tasks" id="menu_project_management_all_tasks" parent="menu_main_pm"
+        <menuitem name="My Tasks" id="menu_project_management_my_tasks" parent="menu_project_management"
+            action="action_server_view_my_task" sequence="1" groups="group_project_user"/>
```

Comment:
Having the group `group_project_user` on these two menuitems is not necessary, as that group is already required to access the app

Reply:
Done changes :+1: